hack/test-cmd.sh

```diff
@@ -327,13 +327,19 @@ for test in "${tests[@]}"; do
   echo
   echo "++ ${test}"
   name=$(basename ${test} .sh)
+  namespace="cmd-${name}"
 
+  os::test::junit::declare_suite_start "cmd/namespace-setup"
```

Comment:
They'll get collapsed down into one when jUnit parsing happens. Can refactor to have distinct setups for each one if you prefer